tests/js/client/shell/shell-iterators.js

```diff
@@ -84,7 +84,7 @@ function IteratorSuite(permuteConfigs) {
   };
 
   return {
-    setUp: function () {
+    setUpAll: function () {
```

Comment:
This is not safe.
There are tests in here that **modify** data in the the collection, e.g.
https://github.com/arangodb/arangodb/pull/17322/files#diff-3588a0693f1c76defd4f9f07238e7e37fca75b717e998352ac9c1fd71f21ca1aR226